<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LocalConfigurationDialog</class>
 <widget class="QDialog" name="LocalConfigurationDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>621</width>
    <height>395</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_4">
   <item row="0" column="0" colspan="2">
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="0">
      <widget class="QCheckBox" name="checkBox_LocalConfigurationInfo">
       <property name="text">
        <string>LocalConfigurationInfo</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QCheckBox" name="checkBox_full">
       <property name="text">
        <string>full</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="lineEdit_full"/>
     </item>
     <item row="1" column="1">
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>LocalConfigurationInfo</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="2" column="0">
           <widget class="QCheckBox" name="checkBox_LocalConfigurationInfoAccessType">
            <property name="text">
             <string>LocalConfigurationInfoAccessType</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QCheckBox" name="checkBox_configuration_key">
            <property name="text">
             <string>configuration_key</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QCheckBox" name="checkBox_configuration_value">
            <property name="text">
             <string>configuration_value</string>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QCheckBox" name="checkBox_failureReason">
            <property name="text">
             <string>failureReason</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QCheckBox" name="checkBox_has_been_updated">
            <property name="text">
             <string>has_been_updated</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="lineEdit_configuration_key"/>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="lineEdit_configuration_value"/>
          </item>
          <item row="3" column="1">
           <widget class="QLineEdit" name="lineEdit_has_been_updated"/>
          </item>
          <item row="4" column="1">
           <widget class="QLineEdit" name="lineEdit_failureReason"/>
          </item>
          <item row="2" column="1">
           <widget class="QComboBox" name="comboBox_LocalConfigurationInfoAccessType"/>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item row="2" column="1">
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QPushButton" name="pushButton_save">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>50</height>
      </size>
     </property>
     <property name="text">
      <string>保存</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="pushButton_quit">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>50</height>
      </size>
     </property>
     <property name="text">
      <string>退出</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
